### PR TITLE
lowered the amount of distance neighbor hits in topoclustering in the ZDC

### PR DESCRIPTION
**Pull request by sebouh137**
### Briefly, what does this PR introduce?
Redefines neighboring subcell hits in the ZDC (in the context of topo clustering) to be the nearest neighbors only for hits in the same layer, and the same position for hits in adjacent layers.   This change reduces the cases where showers from different particles are merged into a single cluster

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [X] Other: parameter tweak

### Please check if this PR fulfills the following:
- [X] Tests for the changes have been added
- [X] Documentation has been added / updated
- [X] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
no.  
### Does this PR change default behavior?
yes.  Changes the default behavior of the topo clustering in the ZDC

### CODE REVIEW

File: src/detectors/ZDC/ZDC.cc [hunk 1]
```diff
@@ -158,12 +158,12 @@ extern "C" {
...
                 .neighbourLayersRange = 1,
-                .localDistXY = {0.76*side_length, 0.76*side_length*sin(M_PI/3)},
-                .layerDistXY = {0.76*side_length, 0.76*side_length*sin(M_PI/3)},
+                .localDistXY = {0.5*side_length, 0.5*side_length*sin(M_PI/3)},
+                .layerDistXY = {0.25*side_length, 0.25*side_length*sin(M_PI/3)},
                 .layerMode=eicrecon::ImagingTopoClusterConfig::ELayerMode::xy,
                 .sectorDist = 10.0 * dd4hep::cm,
                 .minClusterHitEdep = 100.0 * dd4hep::keV,
-                .minClusterCenterEdep = 3.0 * dd4hep::MeV,
+                .minClusterCenterEdep =3.0 * dd4hep::MeV,
```
Comment (ruse-traveler) [inline src/detectors/ZDC/ZDC.cc:166]: Minor nitpick: please keep spacing consistent
```suggestion
                .minClusterCenterEdep = 3.0 * dd4hep::MeV,
```

**Review by ruse-traveler — COMMENTED** [review]
Looks reasonable to me! I have an extremely minor nitpick, but otherwise seems good.

Just so I could educate myself: are there some studies that shows the impact of this tweak? I was having a hard time getting a feel for how this changes things just from the capybara...

**Comment by veprbl** [comment]
@sebouh137 Is this ready? Can you address the review question?

**Comment by sebouh137** [comment]
> @sebouh137 Is this ready? Can you address the review question?

Hi @veprbl I determined that this change was necessary in the context of the Lambda benchmark in the ZDC, 
 https://github.com/eic/physics_benchmarks/pull/19 , since I was seeing a lot of events where clusters were being merged together when they shouldn't be merging (causing a lot of events to have only one cluster, when they should have two or ideally three if the two photons's and the neutron's showers can be distinguished from one another).  I can make some plots with and without this change to the EICrecon

**Review by veprbl — APPROVED** [review]

**Comment by veprbl** [comment]
Feel free to post figures as they become available.

**Pull request merged**